superset-frontend/plugins/plugin-chart-pivot-table/src/plugin/controlPanel.tsx

```diff
@@ -90,15 +89,40 @@ const config: ControlPanelConfig = {
         ],
         ['adhoc_filters'],
         emitFilterControl,
+        ['limit'],
         [
           {
             name: 'row_limit',
             config: {
               ...sharedControls.row_limit,
+              label: t('Cell limit'),
+              description: t('Limits the number of cells that get retrieved.'),
+            },
+          },
+        ],
+        [
```

Comment:
If it doesn't convolute the control panel too much, could we also consider adding `series_columns` to be able to customize which columns to consider in the series limit?

Reply:
Added a TODO comment. We'll add `series_columns` after control panel gets a redesign to avoid clutter